homeassistant/scripts/auth.py

```diff
@@ -25,6 +25,8 @@ def run(args):
     subparsers.required = True
     parser_list = subparsers.add_parser('list')
     parser_list.set_defaults(func=list_users)
+    parser_list.add_argument('-a', '--all', default=False,
+                             help="Show all users included system user")
```

Comment:
This makes no sense. This is a script to manage the `homeassistant` auth provider. System generated users cannot have credentials linked and so will never be listed here